app/views/user_journey/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1 class="govuk-heading-l"><%= t 'components.title' %></h1>
+
```

Comment:
I thought it would be best if we move that around when i do the signing journey as that story will be significantly smaller. Mainly due to the amount of tests that would break and wanting to keep this PR small (well smaller than what it is)

Reply:
yep, good point.. let's push this to the signing journey!